controllers/apps.js

```diff
@@ -92,6 +126,23 @@ module.exports = {
       .catch(error => res.status(400).send(error));
   },
 
+  showUpdatePage(req, res) {
+    return App
+      .findByPk(req.params.appId, {
+        include: [{model: Database, as: 'database'}]
```

Comment:
I don't think we need to include the app's database here since we don't use it on the update page. Might as well save ourselves a query.